SpaceX/app/src/main/java/cz/cvut/popovma1/spacex/feature/rocketlist/presentation/RocketListViewModel.kt

```diff
@@ -6,6 +6,7 @@ import cz.cvut.popovma1.spacex.repository.RocketRepository
 import cz.cvut.popovma1.spacex.repository.model.ResponseWrapper
```

Comment:
Zase mi přijde, že defaultRocketResponse je zbytečná + je na konci souboru a není to přehledné.